x-pack/plugins/security_solution/public/overview/components/detection_response/hooks/use_navigate_to_timeline.tsx

```diff
@@ -33,17 +49,32 @@ export const useNavigateToTimeline = () => {
         providers: [dataProvider],
       })
     );
-    // Only show detection alerts
-    // (This is required so the timeline event count matches the prevalence count)
+
     dispatch(
       sourcererActions.setSelectedDataView({
         id: SourcererScopeName.timeline,
-        selectedDataViewId: 'security-solution-default',
-        selectedPatterns: ['.alerts-security.alerts-default'],
+        selectedDataViewId: defaultDataView.id,
```

Comment:
can you please make this same change in `x-pack/plugins/security_solution/public/common/components/event_details/table/investigate_in_timeline_button.tsx`

Reply:
sorted